### PR TITLE
refactor: s2-react default options and add merge test

### PR DESCRIPTION
**Pull request by yangzhanmei**
### 👀 PR includes

🎨 Enhance

- [ ] Code style optimization
- [x] Refactoring
- [ ] Change the UI
- [ ] Improve the performance
- [ ] Type optimization

🔧 Chore

- [x] Test case
- [ ] Docs / demos update
- [ ] CI / workflow
- [ ] Release version
- [ ] Other ()

### 📝 Description
refactor: s2-react default options and add merge test
fix: s2-react tooltip default options

### CODE REVIEW

File: packages/s2-react/src/common/constant/tooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```
Comment (lijinke666) [inline packages/s2-react/src/common/constant/tooltip.tsx:1]: packages/s2-react/src/common/constant 放在 constant 下面 有点奇怪

Comment (yangzhanmei) [reply packages/s2-react/src/common/constant/tooltip.tsx:1]: 但是它在s2-react层面是算是个option 我也是纠结之后调整到这里的 不然调用的地方都得写一遍🤔

Comment (lijinke666) [reply packages/s2-react/src/common/constant/tooltip.tsx:1]: 确实有点纠结, 是个 option 但是 做了 render 的事情, 放在 src/utils/tooltip.tsx ?

Comment (yangzhanmei) [reply packages/s2-react/src/common/constant/tooltip.tsx:1]: 嗯嗯 也可以 那我换一下

**Review by lijinke666 — APPROVED** [review]

**Pull request merged**